assets/product_code.yml

```diff
@@ -25,3 +25,5 @@ product_code:
   efs-sh19: 40103
   efs-sh19-same-day: 40107
   sr01: 16033
+  confirmation_statement_service_cs01: 15000
```

Comment:
i believe this maps a filing type to a product code, in which case we would want
`confirmation-statement: 15000`
and i suppose `ll-confirmation-statement: 15003`